### ISSUE TITLE
Velocity Sensitivity Overrides Attenuation In Additive Mode

### ISSUE THREAD

**Issue by gdreyband** [bsutherland/JuceOPLVSTi#56]
JUCEOPLVSTi - v14.1
Mac OS - 10.12
Logic Pro X - 10.2.4

When in Additive mode, and velocity sensitivity is set to Off on both M and C, attenuation values work correctly. However when the muted sound has velocity sensitivity set to Light or Heavy, it overrides the value.

Example:

Algo: Additive

Modulator
-47.25
VeloSens: Off

Carrier
0
VeloSens: Off

**Only carrier is audible.**

Algo: Additive

Modulator
-47.25
VeloSens: **Heavy** (or Light)

Carrier
0
VeloSens: Off

**Both Modulator and Carrier are audible.**

**Comment by bsutherland** [comment]
You're right. The sensitivity should really act as a scaling factor for the attenuation setting. Thanks, I'll see what I can do.

**Comment by gdreyband** [comment]
Without being a pain, I'm testing this out in 0.14.2 and I'm not sure the issue was fixed even though it's listed as being addressed in the [release notes](https://github.com/bsutherland/JuceOPLVSTi/releases). 

To clarify from my elaborate example, attenuation works properly with VeloSens set to Off, but seems to be ignored when Light or Heavy. 

It seems to be correctly using the DAW's relative velocity instead. In that case it seems less like a "bug" and more like a design decision. The question becomes do you want VeloSens to be relative to the MIDI volume value alone, or relative to MIDI volume value after attenuation is taken into account?

**Comment by bsutherland** [comment]
Not a pain at all! Really appreciate the QA work ;)

I actually didn't address this one yet though, sorry if I gave that impression in the release notes.

**Comment by gdreyband** [comment]
I misread the issue listed on the release notes. My bad! It's not an insurmountable bug for the time being; just need to set the VeloSens to off.

**Comment by fn0rd23** [comment]
As I remember this behaviour is the same when used in FM config. It seems, when vel. sens. is set to other values than OFF, the attenuation of this operator(s) is not controlled by the attenuation sliders anymore but only by velocity resulting in attenuation of 0dB at full velocity. Maybe it is possible to scale the final velocity value for each operator depending on its current att. slider setting to get relative instead of absolute volume control.

**Comment by bsutherland** [comment]
Yes, it will happen in all modes. I'm planning to implement what you've described.
